examples/some-verifier/src/main.rs

```
@@ -0,0 +1,233 @@
+    let router = Router::new()
+        .route(
+            "/verifications/telegram/:id",
+            get(handle_get_verifications::<Telegram>),
+        )
+        .route(
+            "/verifications/discord/:id",
+            get(handle_get_verifications::<Discord>),
+        )
+        .with_state(state);
+
+    let socket = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), app.api_port);
+    axum::Server::bind(&socket)
+        .serve(router.into_make_service())
+        .await?;
+
+    Ok(())
+}
+
+async fn handle_get_verifications<P: DbPlatform>(
+    State(state): State<AppState>,
+    Path(id): Path<i64>,
+) -> Json<Vec<Verification>> {
+    let accounts = state.database.get_accounts::<P>(id).await;
+    match accounts {
+        Ok(accounts) => {
+            // Futures that simultaneously look up username and revocation status of user
+            // The futures are then mapped to Verifications
+            let futures = accounts.iter().map(|acc| {
+                future::try_join3(
+                    async { Ok(acc.platform) },
+                    get_username(&state, acc),
+                    state
+                        .database
+                        .get_revocation_status(acc)
+                        .map_err(|e| anyhow!(e)),
+                )
+                .map_ok(|(platform, username, revoked)| Verification {
+                    platform,
+                    username,
+                    revoked,
+                })
+            });
+
+            // Keep all the non-error values since the others could not get a username
+            // or a revocation status for whatever reason, probably because the user
+            // does not have that platform
+            let verifications = future::join_all(futures)
+                .await
+                .into_iter()
+                .filter_map(|res| res.ok())
+                .collect();
+            Json(verifications)
+        }
+        Err(_) => Json(vec![]),
```

This does not seem reasonable. Errors should be mapped to errors, they should not be ignored like this  silently.

Database errors will be silently ignored like this.

---

Will be fixed, when #17 is eventually merged.